source/crud/tests/v2/bulkWrite-arrayFilters-clientError.yml

```diff
@@ -0,0 +1,35 @@
+runOn:
+    -
+        # arrayFilters support first introduced in 3.5.6
+        maxServerVersion: "3.5.5"
+
+data:
+    - {_id: 1, y: [{b: 3}, {b: 1}]}
+    - {_id: 2, y: [{b: 0}, {b: 1}]}
+
+collection_name: &collection_name "test"
+database_name: &database_name "crud-tests"
```

Comment:
If this test isn't doing APM assertions (where a collection or database name could show up in the command body), I don't think we need to specify either of these.

Reply:
Quite right.  I forgot to remove them when I trimmed down the file.  They are now gone.